alvr/server_openvr/src/lib.rs

```diff
@@ -407,15 +408,19 @@ pub extern "C" fn shutdown_driver() {
     SERVER_CORE_CONTEXT.write().take();
 }
 
-// Check that there is no active dashboard instance not part of this driver installation
-pub fn should_initialize_driver(driver_layout: &afs::Layout) -> bool {
-    // Note: if the iterator is empty, `all()` returns true
+fn dashboard_process_paths() -> Vec<PathBuf> {
     sysinfo::System::new_all()
         .processes_by_name(OsStr::new(&afs::dashboard_fname()))
-        .all(|proc| {
-            proc.exe()
-                .is_none_or(|path| path == driver_layout.dashboard_exe()) // if path is unreadable then don't care
-        })
+        .filter_map(|proc| Some(proc.exe()?.to_owned()))
+        .collect()
+}
```

Comment:
doing all the work twice is meh, we could just inline this and reuse the gotten result since it's not like a whole lot will shift in the 50 ms between the two uses happening